nuntium/models.py

```diff
@@ -727,16 +746,21 @@ def send_an_email_to_the_author(sender, instance, created, **kwargs):
         if settings.SEND_ALL_EMAILS_FROM_DEFAULT_FROM_EMAIL:
             from_email = settings.DEFAULT_FROM_EMAIL
         else:
+            from_domain = settings.DEFAULT_FROM_DOMAIN
+            if confirmation.message.writeitinstance.config.custom_from_domain:
+                from_domain = confirmation.message.writeitinstance.config.custom_from_domain
             from_email = "%s@%s" % (
                 confirmation.message.writeitinstance.slug,
-                settings.DEFAULT_FROM_DOMAIN,
+                from_domain,
                 )
+        connection = confirmation.message.writeitinstance.config.get_mail_connection()
 
         msg = EmailMultiAlternatives(
             subject,
             text_content,  # content
             from_email,  # From
             [confirmation.message.author_email],  # To
+            connection=connection
```

Comment:
I like final commas on lines like this so that when I come back to it and add another parameter I don't end up giving myself a syntax error!